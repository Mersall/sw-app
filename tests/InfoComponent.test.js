import React from "react";
import renderer from "react-test-renderer";
import InfoComponent from "../component/InfroComponent";

test("renders correctly", async () => {
  const tree = renderer.create(<InfoComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
