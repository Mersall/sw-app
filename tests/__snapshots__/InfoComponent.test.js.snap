// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "column",
      "textAlign": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 30,
        "fontWeight": "600",
        "textAlign": "center",
      }
    }
  />
  <Text>
    There is no episode here
  </Text>
</View>
`;
